### PR TITLE
webide: add vscode settings to disable telementry

### PR DESCRIPTION
**Pull request by bolek-wisniewski**
* Adding settings.json to webide server image, in order to disable
telemetry popup.

* clean webide 400 and 500 error messages (which reveal too much about
software)

* disable daml guidelines link until it can be fixed

* more hiding of content from front end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/docs/source/support/release-notes.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: web-ide/ide-server/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "daml.telemetry": "Disable"
+}
```
Comment (neil-da) [inline web-ide/ide-server/settings.json]: Can we get a trailing newline on this file?

Comment (neil-da) [reply web-ide/ide-server/settings.json]: Did you not also want to reroute some shortcuts?

File: web-ide/proxy/src/routes/webide.ts [hunk 2]
```diff
@@ -41,37 +41,18 @@ export default class WebIdeRoute {
...
-    private proxyError(err :any, req :Request, res :Response) {
-        Session.session(req, res, (err :any, state :any, sessionId :string, saveSession :any) => {
-            console.error("proxy error occurred for session[%s], restarting.", sessionId)
-            this.proxy.removeAllListeners()
-            this.resetState(state, saveSession)
-            this.initProxy()
-        })
-    }
-
+    /* this occurs when  */
```
Comment (neil-da) [inline web-ide/proxy/src/routes/webide.ts]: Truncated comment?

File: web-ide/ide-server/settings.json (hunk 1 above)
Comment (bolek-wisniewski) [reply web-ide/ide-server/settings.json]: thank you for reminding me!

File: web-ide/proxy/src/routes/webide.ts (hunk 2 above)
Comment (bolek-wisniewski) [reply web-ide/proxy/src/routes/webide.ts]: thx

**Pull request merged**